Signal/src/Models/SignalAttachment.swift

```diff
@@ -53,13 +53,31 @@ extension SignalAttachmentError: LocalizedError {
     }
 }
 
-enum TSImageQuality {
-    case uncropped
+@objc
+enum TSImageQuality: UInt {
+    case original
     case high
+    case mediumHigh
     case medium
+    case mediumLow
```

Comment:
There was a steep drop-off in quality in the old arrangement.  I've added more tiers to the quality gradient.